lib/unwrappr/cli.rb

```diff
@@ -68,24 +76,24 @@ def execute
             )
           end
 
-          Dir.chdir(repo) { Unwrappr.run_unwapper_in_pwd(base_branch: base_branch) }
+          Dir.chdir(repo) { Unwrappr.run_unwapper_in_pwd(base_branch: base_branch, lock_files: lock_files) }
         end
       end
     end
   end
 
-  def self.run_unwapper_in_pwd(base_branch:)
-    return unless lockfile_present?
+  def self.run_unwapper_in_pwd(base_branch:, lock_files:)
+    return unless lockfiles_present?(lock_files)
 
     puts "Doing the unwrappr thing in #{Dir.pwd}"
```

Comment:
It would be good to provide some feedback to show that each lockfile has been updated.

Reply:
Good idea. Added in a0ce2ad.